composer.json

```diff
@@ -11,11 +11,11 @@
     ],
     "require": {
         "php": "^7.3 || ^8.0",
-        "friendsofphp/php-cs-fixer": "^3.0",
+        "friendsofphp/php-cs-fixer": "dev-feature/fix-union-types-in-phpdocs as 3.8.0",
```

Comment:
As we discussed, this will make it not installable without copying `repositories` block to every composer.json using `lmc/coding-standard`  😬